apps/meteor/app/lib/server/lib/notifyListenerOnRoomChanges.ts

```diff
@@ -0,0 +1,27 @@
+import { api, dbWatchersDisabled } from '@rocket.chat/core-services';
+import type { IRoom } from '@rocket.chat/core-typings';
+import { Rooms } from '@rocket.chat/models';
+
+type ClientAction = 'inserted' | 'updated' | 'removed';
+
+export async function notifyListenerOnRoomChanges(
+	rid: IRoom['_id'],
+	clientAction: ClientAction = 'updated',
+	existingRoomData?: IRoom,
+	args?: { [key: string]: any },
```

Comment:
if we remove the only place we use this param which is the `avatarETag` one, we may get rid of it..